AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -40,6 +40,8 @@ realty.yandex.ru##.Offer__taxiBannerContainer
 @@||yastatic.net/daas/atom.js$domain=mail.yandex.by|mail.yandex.kz|mail.yandex.ru|mail.yandex.ua|mail.yandex.com|mail.yandex.com.tr
 !-----------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/55984
+t.pia.jp###tg_pia-adFloating
```

Comment:
I would add this rule to the popup.txt, but please don't move it now, wait for another review.

Reply:
All right.

Reply:
self-promo.txt is also proper